src/components/UserProfile/UserIntegrations/index.js

```diff
@@ -69,6 +70,31 @@ const Gen3DetailButton = styled(ConnectedButton)`
   min-width: 190px;
 `;
 
+const ConnectButton = ({ ...props }) => (
+  <ActionButton
+    {...props}
+    css={`
+      padding: 0 6px;
+      border-radius: 19px;
+    `}
+  >
+    <ExternalLinkIcon
+      size={12}
+      css={`
+        margin-left: 10px;
+        margin-right: 5px;
+      `}
+    />
+    Connect
+    <RightIcon
+      size={14}
+      css={`
+        margin-left: 14px;
+      `}
+    />
```

Comment:
I think we prefer splitting these into styled components instead of inline styles now, or styled-system. There's a function called `applyDefaultStyles` you can import from `src/uikit/Core.js` which basically provides all the styled-system functions, so if you wrap the base components with that, you can just use some quick props to do the small styles ;)

Reply:
More like...
```
const ConnectButton = ({ ...props }) => {
  const ExternalLink = applyDefaultStyles(ExternalLinkIcon);
  const RightArrow = applyDefaultStyles(RightIcon);

  return (
    <ActionButton {...props} px={6} br={19}>
      <ExternalLink size={12} ml={10} mr={5} />
      Connect
      <RightArrow size={14} ml={14} />
    </ActionButton>
  );
}
```

Reply:
yeah, or just wrap `applyDefaultStyles` around the `ExternalLinkIcon` and `RightIcon` themselves? personally I think this looks cleaner :P